### PR TITLE
Simple updates

### PR DESCRIPTION
**Pull request by Vladislav94**

### CODE REVIEW

File: src/pages/Main/components/Team/Team.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-import classNames from 'classnames';
 
-import GotchiSvg from 'components/Gotchi/GotchiImage/GotchiSvg';
 import Subtitle from 'components/Subtitle/Subtitle';
 import thegraph from 'api/thegraph.api';
 import hopeUp from 'assets/images/gotchi-placeholder-up.svg';
 
 import styles from './styles';
+import Gotchi from 'components/Gotchi/Gotchi';
+import classNames from 'classnames';
```
Comment (dudendy) [inline src/pages/Main/components/Team/Team.js:11]: please use correct imports order

File: src/components/Icons/Icons.js [hunk 2]
```diff
@@ -154,8 +155,12 @@ export const ListingIcon = ({ width, height }) => {
     return <img width={width} height={height} src={listing} alt='listing' />;
 };
 
-export const LogoIcon = ({ width, height }) => {
-    return <img width={width} height={height} src={logo} alt='logo' />;
+export const LogoIcon = ({ className, width, height }) => {
+    return <img className={className || null} width={width} height={height} src={logo} alt='logo' />;
+};
+
+export const LogoMobile = ({ className, width, height }) => {
```
Comment (dmutrokot) [inline src/components/Icons/Icons.js:162]: `LogoMobile` -> `MobileLogoIcon`

File: src/root/Header/Header.js [hunk 3]
```diff
@@ -41,8 +41,9 @@ export default function Header() {
 
     return (
         <Toolbar className={classes.toolbar}>
-            <NavLink className={classes.logoWrapper} to='/'>
-                <LogoIcon width={80} height={34} />
+            <NavLink className={classes.logoWrapper} to='/' style={{position: 'relative'}}>
```
Comment (dmutrokot) [inline src/root/Header/Header.js:44]: `style={{position: 'relative'}}` please move to the styles file

File: src/root/Header/Header.js [hunk 4]
```diff
@@ -41,8 +41,9 @@ export default function Header() {
...
+                <LogoIcon className={classes.logoDesktop} width={80} height={34} />
+                <LogoMobile className={classes.logoMobile} />
```
Comment (dmutrokot) [inline src/root/Header/Header.js:46]: there are no `width` and `height`, please move width value from styles and set here

**Review by dmutrokot — CHANGES_REQUESTED** [review]

File: src/root/Header/Header.js (hunk 4 above)
Comment (dudendy) [reply src/root/Header/Header.js:46]: and maybe rename it to `<FireballIcon />`

**Review by dmutrokot — APPROVED** [review]

**Pull request merged**